### ISSUE TITLE
TikaFullTextPlugin blows up the entire system

### ISSUE THREAD

**Issue by hsilva-keep** [keeps/roda#229]
*NOTE:* processing an AIP with siard database.
```
[ERROR] [03/02/2016 12:04:52.277] [WorkersSystem-akka.actor.default-dispatcher-9] [akka.actor.ActorSystemImpl(WorkersSystem)] Uncaught error from thread [WorkersSystem-akka.actor.default-dispatcher-9] shutting down JVM since 'akka.jvm-exit-on-fatal-error' is enabled
java.lang.OutOfMemoryError: Java heap space
	at java.util.Arrays.copyOfRange(Arrays.java:3664)
	at java.lang.String.<init>(String.java:207)
	at java.lang.StringBuilder.toString(StringBuilder.java:407)
	at org.apache.solr.handler.loader.XMLLoader.readDoc(XMLLoader.java:416)
	at org.apache.solr.handler.loader.XMLLoader.processUpdate(XMLLoader.java:249)
	at org.apache.solr.handler.loader.XMLLoader.load(XMLLoader.java:177)
	at org.apache.solr.handler.UpdateRequestHandler$1.load(UpdateRequestHandler.java:94)
	at org.apache.solr.handler.ContentStreamHandlerBase.handleRequestBody(ContentStreamHandlerBase.java:69)
	at org.apache.solr.handler.RequestHandlerBase.handleRequest(RequestHandlerBase.java:155)
	at org.apache.solr.core.SolrCore.execute(SolrCore.java:2082)
	at org.apache.solr.client.solrj.embedded.EmbeddedSolrServer.request(EmbeddedSolrServer.java:178)
	at org.apache.solr.client.solrj.SolrRequest.process(SolrRequest.java:149)
	at org.apache.solr.client.solrj.SolrClient.add(SolrClient.java:173)
	at org.apache.solr.client.solrj.SolrClient.add(SolrClient.java:138)
	at org.roda.core.index.IndexModelObserver.indexFile(IndexModelObserver.java:207)
	at org.roda.core.index.IndexModelObserver.fileCreated(IndexModelObserver.java:333)
	at org.roda.core.index.IndexModelObserver.fileUpdated(IndexModelObserver.java:339)
	at org.roda.core.model.ModelObservable.notifyFileUpdated(ModelObservable.java:103)
	at org.roda.core.model.ModelService.updateFile(ModelService.java:1175)
	at org.roda.core.plugins.plugins.ingest.characterization.TikaFullTextPluginUtils.runTikaFullTextOnRepresentation(TikaFullTextPluginUtils.java:83)
	at org.roda.core.plugins.plugins.ingest.characterization.TikaFullTextPlugin.execute(TikaFullTextPlugin.java:109)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.executePlugin(DefaultIngestPlugin.java:481)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.executePlugin(DefaultIngestPlugin.java:467)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.doMetadataAndFullTextExtraction(DefaultIngestPlugin.java:441)
	at org.roda.core.plugins.plugins.ingest.DefaultIngestPlugin.execute(DefaultIngestPlugin.java:255)
	at org.roda.core.plugins.orchestrate.akka.AkkaWorkerActor.onReceive(AkkaWorkerActor.java:46)
	at akka.actor.UntypedActor$$anonfun$receive$1.applyOrElse(UntypedActor.scala:167)
	at akka.actor.Actor$class.aroundReceive(Actor.scala:480)
	at akka.actor.UntypedActor.aroundReceive(UntypedActor.scala:97)
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:526)
	at akka.actor.ActorCell.invoke(ActorCell.scala:495)
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:257)
```

**Comment by luis100** [comment]
Tasks like this need to run on a controlled environment, in this case a different JVM. We need to discuss this further on the next meeting.

**Comment by luis100** [comment]
This will be solved by sandboxing, it is superseeded by #201
